Update TestNG to 7.5.1

abutch3r opened the pull request:
Updated Arquillian related variables to improve consistency and findability across individual projects when updates are to be made.

Update arquillian versions to latest known tested versions that should be compatible with TestNG 7.5.x

7.5.x of TestNG now requires SLF4J, so update ArquillianLoadableExtension to include SLF4J.

fixes #23781, #25109

abutch3r committed:
Update TestNG to 7.5.1

Updated Arquillian related variables to improve consistency and findability across individual projects when updates are to be made.

Update arquillian versions to latest known tested versions that should be compatible with TestNG 7.5.0

7.5.0 of TestNG now includes SLF4J, so update ArquillianLoadableExtension to include SLF4J.

abutch3r commented:
#libby
#build

tevans78 commented on dev/cnf/oss_ibm.maven:
```diff
@@ -70,4 +70,5 @@ org.eclipse.microprofile.rest.client:microprofile-rest-client-tck:1.4.2-20210215
 org.mock-server:mockserver-core:3.10.7-IBM20191022
 org.mock-server:mockserver-core:3.10.7-IBM20191022
 org.openid4java:openid4java:0.9.7-ibm-s20130624-1827
+org.testng:testng:7.5.1
```
Need to move this back to the `oss_dependencies.maven` file now that it is in Maven Central rather than DHE

abutch3r committed:
Resolve testng package references

fix missing slf4j packages in io.openliberty.microprofile.graphql.2.0.internal_fat_tck pom

abutch3r commented:
#libby
#build

abutch3r commented:
dhe check was due to incorrect location of the testng in .dependencies files and has been resolved by restoring to original location

Test failures were due to infrastructure(docker) and a known unrelated test case failure

abutch3r committed:
Add missing SLF4J dependencies

fix missed bnd testng references

abutch3r commented:
#build

abutch3r commented:
#build

abutch3r commented:
#libby

abutch3r commented:
The failing TCK bucket is a intermittent failure on the stopping of the server, all the tests still passed in the bucket

Other failing bucket has since been quarantined in the main build
